src/views/test/DetailPage.tsx

```diff
@@ -0,0 +1,18 @@
+import data from '../customer/CustomerListView/data';
+import { FormikComponent } from './DetailPageCodegen';
+import { Customer } from './Customer';
+
+const CustomerDetailView = () => {
```

Comment:
Let's change the file name - it has too generic name and has different convention than file you have created previously in the same folder `GeneratedCustomers.tsx``. I propose `GeneratedCustomerDetail.tsx` but the component name would be different. Please come up with proper naming... (the same is related also to second component)